### PR TITLE
Add some additional context for opening a PR

### PR DESCRIPTION
**Pull request by schmelto**
I thought it would be good if the Readme shows some more information on how to open a pull request and with `resolve #issunumber` the issues will automatically be closed.

May we also can add a PR Template with `resolve #issuenumber` in it 🚀

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -22,6 +25,10 @@ Feel free to submit a pull request with your suggestions of awesome github profi
 
 **Note** : Please do not edit anything below `<!--START_SECTION:data-section-->` in `README.MD` our [GitHub Action](https://github.com/EddieJaoudeCommunity/gh-action-community) will make the necessary changes for you.
 
+4. Create a PullRequest to add the changes to this Ropository. Please change the title to `add your-username` and in the description add `resolve #issueNumber`
```
Comment (AllanRegush) [inline README.md:28]: I think Pull Request should be two words. Looks like there is a typo in Repository. :nerd_face:

File: README.md [hunk 2]
```diff
@@ -22,6 +25,10 @@ Feel free to submit a pull request with your suggestions of awesome github profi
...
+
+5. The Pullrequest will be approved as soon as possible and afer merge is done you will see you entry in the list
```
Comment (AllanRegush) [inline README.md:30]: I'm not sure if this line is needed? :thinking:

**Review by AllanRegush — COMMENTED** [review]
Looking Good :+1:

**Comment by schmelto** [comment]
@AllanRegush how is it now?

**Comment by AllanRegush** [comment]
@eddiejaoude Should we have the issues close on merge? Users can still vote.

**Review by w3bdesign — APPROVED** [review]

**Comment by AllanRegush** [comment]
I seen in one of the PRs - I don't recall what one., But Eddie said we should be closing the issues :+1:

**Review by AllanRegush — APPROVED** [review]
Thank You

**Comment by eddiejaoude** [comment]
Yeah I think we need to start closing the issues, and historically too, with a note to say, people can still vote

**Pull request merged**